### ISSUE TITLE
Make jcr_root directory configurable

### ISSUE THREAD

**Issue by bobi** [karollewandowski/aem-intellij-plugin#86]
Currently this plugin works only under jcr_root directory, please make this directory configurable, as we are using another directory in our project

**Comment by karollewandowski** [comment]
Thank you for creating this issue. I thought about creating configuration page where user could provide jcr_root paths and patterns. Adding path would be also available through marking directory as JCR root (like *Mark directory as Excluded* etc. provided by IntelliJ Platform).
I'm getting back to this plugin after vacations and will implement improvements. At first I'm going to reduce technical debt by adding lacking tests, so this one have to wait. Contributors are very welcome though ;-)

**Comment by karollewandowski** [comment]
After analysis I decided to implement the following approach:
1. Plugin will detect JCR content directories automatically. Every directory containing `apps`, `etc` or other JCR specific directory with `.content.xml` file (searched recursively to first occurrence) will be automatically marked as JCR content root. User will be able to unmark automatically detected directory.
2. If structure is non-standard, user will be able to mark directory as JCR content root manually (similarly to "Mark as Resources Root", etc).

I started to implement it, but new version will be released after I finish other changes (mostly related to fixing technical debt and improving stability). I expect to finish it in 2-3 weeks.
